.github/workflows/ci.yaml

```diff
@@ -243,7 +243,7 @@ jobs:
       - id: test
         name: Run unit tests
         run: |-
-          docker run -v $(pwd):/python:rw --env CLUSTER_ENV=test --net="host" --entrypoint /usr/bin/env $PYTHON_IMAGE_NAME python3 -m pytest .
+          docker run -v $(pwd):/python:rw --env CLUSTER_ENV=development --net="host" --entrypoint /usr/bin/env $PYTHON_IMAGE_NAME python3 -m pytest .
```

Comment:
Locally, when we run tests, we run them under development, but the ci is configured to use this `test` environment that we don't use. I have changed the CI to use development, but we need to properly clean up this "test" environment or properly introduce it. I will make another ticket for this

Reply:
Ok